lib/capybara/poltergeist/inspector.rb

```diff
@@ -22,9 +22,12 @@ def url
       "//localhost:#{port}/"
     end
 
-    def open
+    def open(scheme = nil)
       if browser
-        Process.spawn(browser, url)
+        scheme = 'http' unless scheme == 'https' # scheme could be 'file' or 'about' or something else
```

Comment:
@twalpole Actually, the scheme argument is more correctly documented as the scheme of the URL being inspected. AFAIK `open` only has one caller and that is what the caller sends. The value only matters to the inspector if it is `https`. It is not invalid or unexpected (rather, it is completely valid *and* expected) to specify other schemes, they just don't matter to the inspector.  There is nothing to warn about.